app-modules/ai/src/Filament/Pages/Assistant/Concerns/CanManageThreads.php

```diff
@@ -333,12 +335,21 @@ public function cloneThreadAction(): Action
                     ->options(function (Get $get): Collection {
                         return match ($get('targetType')) {
                             AiThreadShareTarget::Team->value => Team::orderBy('name')->pluck('name', 'id'),
-                            AiThreadShareTarget::User->value => User::whereKeyNot(auth()->id())->orderBy('name')->pluck('name', 'id'),
+                            AiThreadShareTarget::User->value => User::tap(new WithoutSuperAdmin())->whereKeyNot(auth()->id())->orderBy('name')->pluck('name', 'id'),
                         };
                     })
                     ->searchable()
                     ->multiple()
-                    ->required(),
+                    ->required()
+                    ->rules([
+                        fn (): Closure => function (string $attribute, $value, Closure $fail) {
+                            foreach ($value as $v) {
+                                if (User::findOrFail($v)->hasRole('authorization.super_admin')) {
+                                    $fail('Super admin users cannot have a thread emailed to them.');
```

Comment:
Because this is clone and not email, maybe something like `Super admin users cannot have a thread shared with them.` instead?

Reply:
Done.